app/app.go

```diff
@@ -105,6 +106,8 @@ type TestConfig struct {
 	BroadcastCallback func(context.Context, core.Duty, core.PubKey, core.SignedData) error
 	// DisablePromWrap disables wrapping prometheus metrics with cluster identifiers.
 	DisablePromWrap bool
+	// Enable the builder api
```

Comment:
godoc comments should start with the Field/Function/Type name: `// BuilderAPI enables the builder API opt-in feature`